guides/release/routing/specifying-a-routes-model.md

```diff
@@ -163,11 +175,118 @@ each record in the song model and album model:
 
 If you use [Ember Data](../../models/) and you are building an `RSVP.hash` with the model's relationship, consider instead properly setting up your [relationships](../../models/relationships/) and letting Ember Data take care of loading them.
```

Comment:
If the goal is to focus on the `model` hook, maybe we can remove this sentence?  As a reader, I feel the need to understand how to use `RSVP.hash` with Ember Data after reading this. It also introduce questions about how Ember Data works, and what you mean by building a hash with "the model's relationship" -- even though there have been no mention of model relationship so far in this guide...  I assume that refers to things like `belongs to`  and `have many` ?

Reply:
agreed, removed